Vedant's first contribution (#4)
jedieaston/WinGetHelpers: README.md

# WinGetHelpers!

> __Now updated for 1.0 manifests!__

Reviewer: Please put this message back.

This is a PowerShell module that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs).

## Dependencies

> These are some cmdlets I hastely made to help support microsoft/winget-pkgs. Maybe they'll help you too. 

Reviewer: Please put this message back.

This PowerShell module requires the following dependencies:

> I'm not very good at PowerShell, which is why it looks so bad, but all of these work on my machine(tm) so as long as you are on Windows 10 and have winget installed,

Reviewer: Please put this message back.

they should all work for you too (except Get-GitHubReleases, which requires the GitHub CLI. I'll fix that eventually.) I'll add comments when I get time.
| Dependency | Why it's needed |
| --- | --- |
| Carbon | For getting the Product Code from MSI files |
| powershell-yaml | To parse the YAML files |

# Requirements
You can install both of these at once with the following command:
```
Install-Module powershell-yaml; Install-Module -Name Carbon
```

You'll need two dependencies: [Carbon](http://get-carbon.org/) (for getting the product code from MSI files) and [powershell-yaml](https://github.com/cloudbase/powershell-yaml) (to read and write YAML files). 
## Cmdlets/Functions in this module

> So before you download this module and put it in your modules folder, make sure to do ` Install-Module powershell-yaml; Install-Module -Name 'Carbon' `

Reviewer: Please put this message back. People who want to use this may not be familiar with PowerShell.

Reviewer 2: I wrote something similar here.

https://github.com/vedantmgoyal2009/WinGetHelpers/blob/patch-1/README.md?plain=1#L14-L17

Reviewer 2: https://github.com/jedieaston/WinGetHelpers/pull/4/commits/78f3a48c1144055b2095b5d0695ab9d05998fe8a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18-R21

Author: https://github.com/vedantmgoyal2009/WinGetHelpers/blob/patch-1/README.md?plain=1#L18-L21

Reviewer 2: Please refer to the last one. The above two are outdated.

# WinGetHelpers!

> __Now updated for 1.0 manifests__

Reviewer: ```suggestion
__Now updated for 1.0 manifests!__
```

> I hastily made this PowerShell module, that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs). Maybe it will be useful for other people too.

Reviewer: ```suggestion
I hastily made this PowerShell module, that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs). Maybe it will be useful for you too.
```
